welcome/oke_about.adoc

```diff
@@ -286,6 +286,7 @@ s| Feature s| {oke} s| {product-title} s| Operator name
 | Embedded OperatorHub | Included | Included | N/A
 | Embedded Marketplace | Included | Included | N/A
 | Quay Compatibility (not included) | Included | Included | N/A
+| OpenShift API for Data Protection (OADP) | Included | Included | OADP Operator
```

Comment:
So far I only added it to the comprehensive list of features, but if OADP should also be added to [this seemingly higher-level list](https://docs.openshift.com/container-platform/4.16/welcome/oke_about.html#about_oke_similarities_and_differences) of similarities and differences, let me know.

Reply:
I like it the way it is, w/o the higher level list.